devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -111,9 +111,8 @@ public void setSourceDir(String sourceDir) {
         this.sourceDir = sourceDir;
     }
 
-    @Option(description = "Set working directory", option = "working-dir")
-    @Deprecated
     @Input
+    @Deprecated
     // @InputDirectory this breaks kotlin projects, the working dir at this stage will be evaluated to 'classes/java/main' instead of 'classes/kotlin/main'
```

Comment:
Looks like you got it right! Probably a path issue on windows?
If it's fully working, perhaps, we shouldn't be deprecating the method.

Reply:
@aloubyansky you are right, this is a path issue on windows, I updated the code to use `java.io.tmpdir` instead. 
Using the getter to get `workingDir` instead of accessing it directly fix the issue.